model_analyzer/config/generate/run_config_generator_factory.py

```diff
@@ -55,7 +60,9 @@ def create_run_config_generator(
         models: List[ConfigModelProfileSpec],
         client: TritonClient,
         result_manager: ResultManager,
+        metrics_manager: MetricsManager,
```

Comment:
Why is the MetricsManager now needed by an RCG? Why are all of these things needed by an RCG?